packages/components/src/Disclosure/docs/disabled.tsx

```diff
@@ -0,0 +1,20 @@
+import { Disclosure, DisclosureHeader, DisclosurePanel, Div, Text } from "@hopper-ui/components";
+import { SparklesIcon } from "@hopper-ui/icons";
+
+export default function Example() {
+    return (
+        <Div width="100%" paddingX="core_320" paddingY="core_480">
+            <Disclosure isDisabled>
+                <DisclosureHeader prefix={<SparklesIcon />}>
```

Comment:
Why is there a prefix props? it should be a slot no?

Reply:
just like the buttons

Reply:
I tried it as a slot, but there was a big issue with the styling and how the wrapping of title and description should behave. Trust me, I tried all things, but we needed a wrapper around title and description.